### ISSUE TITLE
Add wp-cli support

### ISSUE THREAD

**Issue by LinuxJedi** [MariaDB/wordpress-mariadb-health-checks#31]
A user should be able to do things such as histograms via wp-cli

**Comment by scottbuscemi** [comment]
Would love this as well. Ran into site timeouts when using the dashboard.

**Comment by LinuxJedi** [comment]
> Would love this as well. Ran into site timeouts when using the dashboard.

Good to know, was this when using the histograms feature?

**Comment by scottbuscemi** [comment]
Yes, it seems they went through successfully but the web server timed out the request on the front end.

**Comment by LinuxJedi** [comment]
Makes sense, I'll work on this ASAP and make it a priority for the major release after 1.1.

**Comment by scottbuscemi** [comment]
Thanks! Happy to connect you with [Pantheon](https://pantheon.io) engineers if we can help at all with how to configure the CLI commands.
